### PR TITLE
Fixes/updates most of the Javascript + Fixes dialogs not working and removes size column

### PR DESCRIPTION
**Pull request by OmgImAlexis**

### CODE REVIEW

**Comment by MGaetan89** [comment]
You left some `bind` methods ;)

**Comment by OmgImAlexis** [comment]
The ones I left seem to work, I'll change the rest of them tonight as I'm doing a huge overhaul of the js then.

**Comment by MGaetan89** [comment]
Maybe `Move home.js into file and 'fix' size column` should be in an other PR.
So it will be easier in the future to find again and revert the PR that hides the show size column.

**Comment by OmgImAlexis** [comment]
@MGaetan89 will do.

File: gui/slick/views/home.mako [hunk 1]
```diff
@@ -41,353 +41,19 @@
...
+<meta data-var="sickbeard.POSTER_SORTBY" data-content="${sickbeard.POSTER_SORTBY}">
+<meta data-var="sickbeard.POSTER_SORTDIR" data-content="${sickbeard.POSTER_SORTDIR}">
+<meta data-var="sickbeard.FUZZY_DATING" data-content="${sickbeard.FUZZY_DATING}">
+<meta data-var="sickbeard.DATE_PRESET" data-content="${sickbeard.DATE_PRESET}">
+<meta data-var="sickbeard.TIME_PRESET" data-content="${sickbeard.TIME_PRESET}">
+<meta data-var="sickbeard.TRIM_ZERO" data-content="${sickbeard.TRIM_ZERO}">
+<meta data-var="max_download_count" data-content="${max_download_count}">
+<meta data-var="layout" data-content="${layout}">
+<meta data-var="fuzzydate" data-content="${fuzzydate}">
+<script type="text/javascript" src="${sbRoot}/js/new/home.js"></script>
```
Comment (OmgImAlexis) [inline gui/slick/views/home.mako]: For those wondering, for now we're using meta tags to store information we need in js until the v2 api gets view paths added.

**Pull request closed without merging**